common/app/layout/Breakpoint.scala

```diff
@@ -0,0 +1,19 @@
+package layout
+
+import scalaz.syntax.std.option._
+
+sealed trait Breakpoint {
+  val minWidth: Option[Int]
+}
+
+case object Mobile extends Breakpoint {
+  val minWidth = None
+}
+
+case object Tablet extends Breakpoint {
+  val minWidth = 740.some
```

Comment:
Ah yeah, you're right. :+1: 

Please don't kill me, Rich :sob: I'll do better!